src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js

```diff
@@ -4,42 +4,49 @@ import { withStyles } from "@material-ui/styles";
 import ListView from "./ListView";
 import StyledCard from "./StyledCard";
 
+import CardImg from "../../../../../assets/images/dummy-card.png";
+
 import { useStyles } from "./styles";
 
 class CardGroup extends Component {
```

Comment:
It doesn't have any state, so better keep it functional component.

After converting it into functional components, check the styling once. coz useStyles have to be a hook, then.